ee/ucp/admin/install/install-on-azure.md

```diff
@@ -4,25 +4,107 @@ description: Learn how to install Docker Universal Control Plane in a Microsoft
 keywords: Universal Control Plane, UCP, install, Docker EE, Azure, Kubernetes
 ---
 
+Docker UCP closely integrates into Microsoft Azure for its Kubernetes Networking 
+and Persistent Storage feature set. UCP deploys the Calico CNI provider for data path networking and leverages Azure CNI for IP address management. There are infrastructure prerequisites that are required prior to UCP installation for the Azure IPAM functionality.
+
+
+## Docker UCP Networking
+
 Docker UCP configures the Azure IPAM module for Kubernetes to allocate
-IP addresses to Kubernetes pods. The Azure IPAM module requires each Azure
+IP addresses to Kubernetes pods.  The Azure IPAM module requires each Azure
 VM that's part of the Kubernetes cluster to be configured with a pool of
 IP addresses.
 
 You have two options for deploying the VMs for the Kubernetes cluster on Azure:
 - Install the cluster on Azure stand-alone virtual machines. Docker UCP provides
   an [automated mechanism](#configure-ip-pools-for-azure-stand-alone-vms)
-  to configure and maintain IP pools for stand-alone Azure VMs.
+  to configure and maintain IP pools for stand-alone Azure VMs. 
 - Install the cluster on an Azure virtual machine scale set. Configure the
-  IP pools by using an ARM template like [this one](#set-up-ip-configurations-on-an-azure-virtual-machine-scale-set).
+  IP pools by using an ARM template like 
+  [this one](#set-up-ip-configurations-on-an-azure-virtual-machine-scale-set).
 
 The steps for setting up IP address management are different in the two
 environments. If you're using a scale set, you set up `ipConfigurations`
 in an ARM template. If you're using stand-alone VMs, you set up IP pools
 for each VM by using a utility container that's configured to run as a
 global Swarm service, which Docker provides.
 
-## Considerations for size of IP pools
+## Azure Prerequisites 
+
+The following list of infrastructure prerequisites need to be met in order 
+to successfully deploy Docker UCP on Azure.
+
+- All UCP Nodes (Managers and Workers) need to be deployed into the same 
+Azure Resource Group. The Azure Networking (Vnets, Subnets, Security Groups) 
+components could be deployed in a second Azure Resource Group.
+- All UCP Nodes (Managers and Workers) need to be attached to the same 
+Azure Subnet.
+- All UCP (Managers and Workers) need to be tagged in Azure with the
+tag `Orchestrator=Kubernetes:1.8.11`.
```

Comment:
The tag `1.8.11` in the docs will need to be kept up-to-date with every UCP patch release where kube minor patch versions are updated. To make this documentation more applicable across different versions of UCP (which will deploy different versions of Kube), please make this more generic and explain with examples.